sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -460,14 +460,6 @@
                 "value": "studentfinancialinformationappeal",
                 "label": "Financial Information"
               },
-              {
```

Comment:
Please also remove the variables in workflow load-assessment-consolidated-data. 

![image](https://github.com/user-attachments/assets/3f1a01e5-7dad-4f4b-a583-99311dfe6532)

Reply:
Thanks for the explanation @dheepak-aot